cmd/generate_repo_config/generate_repo_config_test.go

```diff
@@ -104,3 +104,73 @@ go_repository(
 		},
 	})
 }
+
+func TestDuplicateRepo(t *testing.T) {
```

Comment:
NIT: This test and the one above it seem to have a very similar structure, they could probably be merged into a table test.